addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -414,11 +419,6 @@
                 </xpath>
                 <xpath expr="//div[hasclass('alert-info')]/div[hasclass('o_mails_in_queue')]" position="replace"/>
                 <xpath expr="//div[hasclass('oe_button_box')]" position="inside">
-                    <button name="action_view_traces_sent"
```

Comment:
Yes I can see that, but it means that it is now placed in "view_mail_mass_mailing_form" instead of "mailing_mailing_view_form_full_width", is that really intended? 

If it's just to move the button around, you could have simply adapted the xpath expression here.

Reply:
oh, my bad done with it.